src/tasks/migrations/rollback.js

```diff
@@ -6,15 +6,42 @@ const {
   urlTofRollbackMigrationFile
 } = require('./utils')
 
+/**
+ * @method isStoryPublishedWithoutChanges
+ * @param  {Object} story
+ * @return {Boolean}
+ */
+const isStoryPublishedWithoutChanges = story => {
+  return story.published && !story.unpublished_changes
+}
+
+/**
+ * @method isStoryWithUnpublishedChanges
+ * @param  {Object} story
+ * @return {Boolean}
+ */
+const isStoryWithUnpublishedChanges = story => {
+  return story.published && story.unpublished_changes
+}
+/**
+ * @typedef {'all'|'published'|'published-with-changes'} PublishOptions
+ *
+ * @typedef {Object} RunRollbackOptions
+ * @property {PublishOptions} publish
+ * /
+
```

Comment:
These functions already exist in the `run.js` file, how about extracting them from there to a constants file and using them in both places?

Reply:
Done, moved them to `utils.js` file and reused them in `run.js` and `rollback.js` files.